src/components/cards/LunchCard.vue

```diff
@@ -5,34 +5,99 @@
           No Lunch Data
          <what-is-this>Stevenson.Space relies on quality data dispayed by Stevenson's lunch website. We will promptly restore lunch functionality once data is provided there.</what-is-this>
     </div>
-    <div v-else v-for="(items, name) in lunch" :key="name" class="lunch">
-      <div class="name">{{ name }}</div>
-      <div v-for="item in items" :key="item" class="item">
-        {{ item }}
+    <div v-else>
+      <div v-if="isMorning" class="lunch">
+        <div class="name">Ala Carte Breakfast</div>
+        <div v-for="item in alBreakfast" :key="item" class="item">
+          {{ item }}
+        </div>
+      </div>
+      <div v-for="(items, name) in lunch" :key="name" class="lunch">
+        <div class="name">{{ name }}</div>
+        <div v-for="item in items" :key="item" class="item">
+          {{ item }}
+        </div>
+      </div>
+      <div v-for="(items, name) in mData" :key="name" class="exLunch">
+        <div class="name">{{ name }}</div>
+        <div v-for="item in items" :key="item" class="item">
+          {{ item }}
+        </div>
       </div>
     </div>
+    <p>{{isExpanded}}</p>
```

Comment:
isExpanded is not a variable used anywhere, it looks like it was used for debugging or something